cfgov/hmda/jinja2/hmda/hmda-explorer-institutions.html

```diff
@@ -13,63 +13,64 @@ <h3>Related nationwide institution data</h3>
-                        <td data-label="Panel"><a href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2009_panel.csv">CSV</a> (381 KB)</td>
+                        <td data-label="Transmittal sheet"><a aria-label="2009 transmittal sheet CSV file" href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2009_transmittal_sheet.zip">CSV</a> (413 KB)</td>
+                        <td data-label="Panel"><a aria-label="2009 panel CSV file" href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2009_panel.zip">CSV</a> (381 KB)</td>
                     </tr>
                     <tr>
                         <td data-label="Year">2008</td>
-                        <td data-label="Transmittal sheet"><a href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2008_transmittal_sheet.csv">CSV</a> (428 KB)</td>
-                        <td data-label="Panel"><a href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2008_panel.csv">CSV</a> (393 KB)</td>
+                        <td data-label="Transmittal sheet"><a aria-label="2008 transmittal sheet CSV file" href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2008_transmittal_sheet.zip">CSV</a> (428 KB)</td>
+                        <td data-label="Panel"><a aria-label="2008 panel CSV file" href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2008_panel.zip">CSV</a> (393 KB)</td>
                     </tr>
                     <tr>
                         <td data-label="Year">2007</td>
-                        <td data-label="Transmittal sheet"><a href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2007_transmittal_sheet.csv">CSV</a> (440 KB)</td>
-                        <td data-label="Panel"><a href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2007_panel.csv">CSV</a> (404 KB)</td>
+                        <td data-label="Transmittal sheet"><a aria-label="2007 transmittal sheet CSV file" href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2007_transmittal_sheet.zip">CSV</a> (440 KB)</td>
+                        <td data-label="Panel"><a aria-label="2007 panel CSV file" href="https://files.consumerfinance.gov/hmda-historic-institution-data/hmda_2007_panel.zip">CSV</a> (404 KB)</td>
                     </tr>
                 </tbody>
             </table>
         </div>
-        <p class="u-mt10">See the documentation for the <a href="#">transmittal sheet</a> and <a href="#">panel data</a>.</p>
+        <p class="u-mt10 u-mb5">Panel data documentation: <a href="https://www.ffiec.gov/hmdarawdata/FORMATS/2009HMDAReporterPanel.pdf">2007-2009</a> and <a href="https://s3.amazonaws.com/cfpb-hmda-public/prod/snapshot-data/2017_publicstatic_dataformat.zip">2010-2017</a>.</p>
```

Comment:
How would you feel about hosting these PDFs on files.consumerfinance.gov? That way we won't need to depend on ffiec, and we can give them more sensible names.

Reply:
👍